Changes Command departmental channel back to gold

AnonymousNow opened the pull request:
_Changes the Command departmental radio channel's colour from its current navy back to gold (a darker one than before)_, changing a similar tweak in November. Images attached; edited together from uraniummeltdown's PR, which added the Bay colours, showing the original gold, navy and Medbay blue, plus one of my own that shows the darker gold. Thanks, MSO.
![command3](https://cloud.githubusercontent.com/assets/6266927/22859185/58586fa4-f0cb-11e6-88e8-58be1059c45c.png)
![comnewoldmed](https://cloud.githubusercontent.com/assets/6266927/22859186/585a272c-f0cb-11e6-88e7-06853cc04f85.png)

It suits the setting better (was blue/navy ever particularly associated with command? It was usually green or gold, unless you're focusing on the captain's outfit), and makes it more distinct from Medical's lighter blue. If I have trouble telling the difference on the fly, with my above-average colour vision and relatively high saturation setting on my screen, then what've other people been doing?

AnonymousNow committed:
Changes Command departmental channel back to gold

I don't know how Github works and I'm likely about to break /everything/ in a domino effect cataclysm the likes of which no-one has ever seen. But if this actually works, then what does that say for the future of humanity?

Changes the Command departmental radio channel's colour from its current navy back to its original gold, reverting a change in November. It suits the setting better (was blue/navy ever associated with command? It was usually green or gold), and makes it more distinct from Medical's lighter blue. If I have trouble telling the difference on the fly, with my above-average colour vision and relatively high saturation setting on my screen, then what've other people been doing?

dannno commented:
the blue is nicer

AnonymousNow commented:
@dannno I'm a sucker for a good blue, but I think simple readability is more important here. Chat is noisy enough as it is.

chanoc1 commented:
no you are angry code man go back to lifeweb

Anonmare commented:
I can't read gold on white without squinting so :-1: from me

AnonymousNow commented:
Alright, can you suggest a better distinct colour for Command that's easier to see?

GunHog commented:
Pics plz

Anonmare commented:
@AnonymousNow Probably dark gold or anything that contrasts against a white background. It's an oft-used channel so quick recognition and readability is important, I remember hating playing Command because the yellow it used blended in with the white too much.

AnonymousNow commented:
@GunHog Added.
@Anonmare That makes sense. This is an attempted readability change, so I'd like to make sure that it's as distinct as possible for as many people as possible. I'll have a look at other shades now.

GunHog commented:
Could we have a darker gold?

PKPenguin321 commented:
I think this is important if only for the colorblind. If you can't read this shade of yellow on white, your monitor settings are probably fucked.

optimumtact commented:
The blue is nicer

MrStonedOne commented:
http://www.rapidtables.com/web/color/Gold_Color.htm

AnonymousNow committed:
Changes Command departmental channel back to gold (darker)

Changes the Command departmental radio channel's colour from its current navy back to (a darker) gold, changing a pull in November. Image attached; edited together from uraniummeltdown's PR, which added the Bay colours, showing the gold, navy and Medbay blue.

It suits the setting better (was blue/navy ever particularly associated with command? It was usually green or gold, unless you're focusing on the captain's outfit), and makes it more distinct from Medical's lighter blue. If I have trouble telling the difference on the fly, with my above-average colour vision and relatively high saturation setting on my screen, then what've other people been doing?

AnonymousNow commented:
@Anonmare I've picked out a darker, more vivid gold. If I knew how to preview it ingame quickly, I would, but the lorem ipsum here should demonstrate.

It stands out more, verges on green a little bit. How's that look?

Anonmare commented:
@AnonymousNow Yeah I can read that without having to scratch my eyes out

MrStonedOne commented:
download link:

https://github.com/AnonymousNow/tgstation/archive/patch-1.zip

Make sure you have 511  version of byond installed, just open dream maker, point it to the tgstation.dme file, then go to build -> clean compile (hit yes)

then open dream deamon and point it to the tgstation.dmb file (the one with more green in the icon)
set it to trusted mode and then press go (once, don't double click).

after 5 to 60 seconds, the go will turn to stop, and above it will be a yellow road sign icon, click it to join the server.

kevinz000 commented:
I've made this PR before but it was rejected. Thanks alot oranges.

kevinz000 commented:
https://github.com/tgstation/tgstation/pull/21596

uraniummeltdown commented:
No way keep it blue it's the same color as the Command airlocks

AnonymousNow commented:
Thanks @MrStonedOne ; I put my own colour up.
@kevinz000 Glad I'm not the only one with this issue.

XDTM commented:
Yeah the blue for command comes from the airlocks, i'm not sure what's the reasoning behind gold except maybe for captain's ID.

PKPenguin321 commented:
Colorblindness

Dorsisdwarf commented:
👍  as fuck

AnonymousNow commented:
Well, now what? How can I chase this up? It's in the state it needs to be in, as far as I'm aware.

PKPenguin321 commented:
Now you just wait

Unfortunately citrus was memed to death so it will probably sit for a few days while the other maintainers do ???

AnonymousNow commented:
Cool. Let's see if I can contribute more.

Qbopper commented:
absolutely not in favour of this, coming back and finding command to be a nice blue was the best and now it's back to yellow on white memes

AnonymousNow commented:
Well, it's a readability fix. D'ye think I'd change something from blue if it wasn't important? Me?

PKPenguin321 commented:
Qbopper listen
colorblindness is the reason

Qbopper commented:
screw the disabled i'm more important

and yeah I dunno I just think gold is a poor choice for a colour - it may make things easier for those with colourblindness but imo it also makes it harder for people without colourblindness to read

it's not a big deal either way, I was trying to shitpost
